product_docs/docs/tde/15/secure_key/disabling_key.mdx

```diff
@@ -13,3 +13,7 @@ If you don't want key wrapping, for example, for testing purposes, you can use e
 With either one of the configurations, TDE generates encryption key files, but leaves them unprotected.
 
 For `intidb --data-encryption` to run successfully, you have to either specify a wrapping/unwrapping command, set a fallback environment variable with wrapping/unwrapping commands, or disable key wrapping with the one of the previous mechanisms. Otherwise, the creation of an encrypted database cluster will fail.
+
+!!!note
+   If you want to enable key wrapping on TDE-enabled database clusters where key wrapping was previously disabled, see [Enabling a mechanism to protect the data encryption key](../enabling/enabling_key_wrapper).
+!!!
```

Comment:
There's an admonition warning here because I think the actual content of the note above is indented and that makes it unfenced and then the checker sees a !!! after it and that looks like the start of another admonition. Not an issue on this pass but if text is added after this, it may light up as an admonition.